recipe/meta.yaml

```diff
@@ -32,25 +28,20 @@ requirements:
     - make  # [unix]
     - python  # [win]
   host:
-    - boost-cpp
     - gtest
     - gmock
     - zlib
-  run:
-    - boost-cpp
 
 test:
   commands:
     - test -f $PREFIX/include/msgpack.h                         # [unix]
-    - test -f $PREFIX/include/msgpack.hpp                       # [unix]
-    - test -f $PREFIX/lib/libmsgpackc.a                         # [unix]
-    - test -f $PREFIX/lib/libmsgpackc.so                        # [linux]
-    - test -f $PREFIX/lib/libmsgpackc.dylib                     # [osx]
+    - test -f $PREFIX/lib/libmsgpack-c.a                        # [unix]
```

Comment:
```suggestion
    - test ! -f $PREFIX/lib/libmsgpack-c.a                        # [unix]
```

Reply:
Ok, that make sense as per [CFEP-018](https://github.com/conda-forge/cfep/blob/main/cfep-18.md). However, probably I can do the same for Windows for consistency?

Reply:
I'm just not as adept with windows. So I am not able to provide stronger recommendations there. But yes, doing the same for windows would be preferred.